src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -299,13 +299,47 @@ public void SendFunctionLoadRequests(ManagedDependencyOptions managedDependencyO
                     _eventSubscriptions.Add(_inboundWorkerEvents.Where(msg => msg.MessageType == MsgType.FunctionLoadResponse)
                         .Subscribe((msg) => LoadResponse(msg.Message.FunctionLoadResponse), HandleWorkerFunctionLoadError));
                 }
-                foreach (FunctionMetadata metadata in _functions.OrderBy(metadata => metadata.IsDisabled()))
+
+                // Check if the worker supports this feature
+                bool capabilityEnabled = !string.IsNullOrEmpty(_workerCapabilities.GetCapabilityState(RpcWorkerConstants.AcceptsListOfFunctionLoadRequests));
```

Comment:
With this logic, the worker should not send "false" and expect it to be disabled. correct?

Reply:
You may also consider using [IsNullOrWhiteSpace](https://docs.microsoft.com/en-us/dotnet/api/system.string.isnullorwhitespace?redirectedfrom=MSDN&view=net-6.0#remarks)

Reply:
yes